src/components/views/dialogs/ExportDialog.tsx

```diff
@@ -361,6 +364,7 @@ const ExportDialog: React.FC<IProps> = ({ room, onFinished }) => {
                     />
 
                     <StyledCheckbox
+                        id="include-attachments"
```

Comment:
we seem to be using a mix of `data-test-id` and `id` - is there a documentable difference between the two? (if so, can we document it please?)

Reply:
Used `id` on the inputs because otherwise `Field` generates an id that messes with the snapshot tests. Since these ids already make them easy to select I didn't bother adding `data-test-id`s here too.
Otherwise I've used `data-test-id` because their purpose is clear (testing) and they don't need to be unique across the document. Not really sure this is worth documenting